### PR TITLE
feat: Support getting an organization role

### LINKED ISSUE #3282: feat: Support getting an organization role
Support getting an organization role. [Docs here](https://docs.github.com/en/enterprise-cloud@latest/rest/orgs/organization-roles?apiVersion=2022-11-28#get-an-organization-role)

I can also work on this

### PR DESCRIPTION
**Pull request by felixlut**
Resolves #3282 

It was only possible to list organization roles before, this makes it possible to get a specific one. 

I might have went overboard with the test-cases (testing both a custom role and a built-in one). I haven't written mocks before, so I'm not sure what the best practices are here 😅

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @felixlut !
LGTM.

Awaiting second LGTM+Approval from any other contributor to this repo before merging.

**Review by tomfeigin — APPROVED** [review]
LGTM

**Comment by gmlewis** [comment]
Thank you, @tomfeigin !
Merging.

**Pull request merged**
